tests/Unit/ProxyClient/NoopTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of the FOSHttpCache package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\HttpCache\Tests\Unit\ProxyClient;
+
+use FOS\HttpCache\ProxyClient\Invalidation\BanInterface;
+use FOS\HttpCache\ProxyClient\Noop;
+
+class NoopTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var Noop
+     */
+    private $noop;
+
+    protected function setUp()
+    {
+        $this->noop = new Noop();
+    }
+
+    public function testBan()
+    {
+        $this->assertInstanceOf(BanInterface::class, $this->noop->ban(['header-123']));
```

Comment:
Totally agree here, my IDE picked up `$this` as that and as I like a train monkey I just did it. Will sort later today.